src/linear_algebra/matrix/bilinear_form.lean

```diff
@@ -0,0 +1,556 @@
+/-
+Copyright (c) 2018 Andreas Swerdlow. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andreas Swerdlow, Kexing Ying
```

Comment:
It looks to me like @Vierkantor is actually mostly responsible for this file (#2700, #5095), and that Andreas's original code didn't contain anything that's here. So maybe
```suggestion
Copyright (c) 2020 Anne Baanen. All rights reserved.
Released under Apache 2.0 license as described in the file LICENSE.
Authors: Anne Baanen, Kexing Ying
```

It's entirely plausible to me that @JasonKYi was involved in this file, so I'm happy to leave them in.